backend/api/Controllers/PropertyController.cs

```diff
@@ -88,5 +86,45 @@ public IActionResult GetProperties([FromBody]PropertyFilterModel filter)
             return new JsonResult(properties.ToArray());
         }
         #endregion
+
+
+        #region Property List View Endpoints
+        /// <summary>
+        /// Get all the properties that satisfy the filter parameters.
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet("page")]
+        [HasPermission(Permissions.PropertyView)]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(PageModel<PropertyModel>), 200)]
+        [SwaggerOperation(Tags = new[] { "property" })]
+        public IActionResult GetPropertiesPage()
+        {
+            var uri = new Uri(this.Request.GetDisplayUrl());
+            var query = Microsoft.AspNetCore.WebUtilities.QueryHelpers.ParseQuery(uri.Query);
+            return GetPropertiesPage(new PropertyFilterModel(query));
+        }
+
+        /// <summary>
+        /// Get all the properties that satisfy the filter parameters.
+        /// </summary>
+        /// <param name="filter"></param>
+        /// <returns></returns>
+        [HttpPost("page/filter")]
+        [HasPermission(Permissions.PropertyView)]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(PageModel<PropertyModel>), 200)]
+        [SwaggerOperation(Tags = new[] { "property" })]
+        public IActionResult GetPropertiesPage([FromBody]PropertyFilterModel filter)
+        {
+            filter.ThrowBadRequestIfNull($"The request must include a filter.");
+            if (!filter.IsValid()) throw new BadRequestException("Property filter must contain valid values.");
+
+            var page = _pimsService.Property.Get((AllPropertyFilter)filter);
```

Comment:
minor naming comment - shouldn't the above be named `service.Property.GetPage(...)` for consistency with the other "page" methods?